packages/block/src/types.ts

```diff
@@ -89,6 +68,12 @@ export interface BlockData {
   uncleHeaders?: Array<HeaderData>
 }
 
+export type BlockBuffer = [BlockHeaderBuffer, TransactionsBuffer, UncleHeadersBuffer]
+export type BlockHeaderBuffer = Buffer[]
+export type BlockBodyBuffer = [TransactionsBuffer, UncleHeadersBuffer]
+export type TransactionsBuffer = Buffer[][]
+export type UncleHeadersBuffer = Buffer[][]
```

Comment:
Yes, totally makes sense to have these types.

Reply:
:) I tried to do some fancy array unpacking with the spread operator to make it look more compact (e.g. `type BlockBuffer = [BlockHeaderBuffer, ...BlockBodyBuffer]` but ts was not liking it so I went for this little more verbose route.